client/.eslintrc.js

```diff
@@ -26,5 +26,6 @@ module.exports = {
   "rules": {
       'react/prop-types': ['off'],
       'react/display-name': ['off'],
+      "no-debugger":"off",
```

Comment:
I added this so eslint would stop complaining about debugger statements when I was debugging.